### DISCUSSION TITLE
the PFT conversion problems

### DISCUSSION THREAD

**Discussion by niuhanlin** [NGEET/fates#924] [Q&A]
In CLM, the PFT of cool grass was 13, and when converted to FATES, it was 11.
But  it was identified as 10 in the simulation results. I wonder if this is correct?
![fates_hlm_pft_map_in_fates_params_api 24 0 0_12pft_c220608_ 5](https://user-images.githubusercontent.com/66841974/196608953-431355d9-7a18-4848-bbd4-0a0b07f64bab.png)

![FATES_GPP_PF_in_maqu_grass_fix_biogeography_1 clm2 h0 2012](https://user-images.githubusercontent.com/66841974/196608789-5391c280-11b3-4e5e-b2fb-b6db447b8d54.png)

**Comment by rosiealice** [comment]
Is the mismatch between 10 and 11 something to do with python zero indexing issues?

The conversion from CLM to fates parameters I expected, as they do not map 1:1, and instead are converted via the hlm_to_fates pft index. I'll try and find the exact name for it.

**Reply by rosiealice** [reply]
It's fates_hlm_pft_map 

This is an array, so the area of one HLM pft can be mapped onto >1 fates PFTs, or vice versa.

**Comment by niuhanlin** [comment]
@rosiealice 
Yes, I used fates_hlm_pft_map to determine the conversion of PFT from CLM to FATES.
The local grass is cool grass, which I set to 100% in the surface file. It all makes sense until I run it.

But in use_nl_clm I set the output variable: FATES_GPP_PF to try to discern the output pft result.
My expected PFT should be cool grass(C3), whose PFT in FATES is 11.
But the result is Arctic grass (PFT 10), as mentioned above in the picture below.
